DEVELOPING_OPAL/validate/src/it/scala/org/opalj/br/reader/InvokedynamicRewritingBytecodeStructureTest.scala

```diff
@@ -165,6 +165,25 @@ class InvokedynamicRewritingBytecodeStructureTest extends AnyFunSpec with Matche
             }
         }
 
+        describe("testing the rewritten methods of the java17sealedclass test project") {
```

Comment:
Since we do not perform any specific rewriting for sealed classes, I don't think we need this test.